### PR TITLE
theming: Frontpage layout

### PR DESCRIPTION
**Pull request by sebastiendeleze**
* NEW HTML + CSS application layout.
* NEW Add Bootstrap 4 library.
* NEW Add Google Roboto font.
* BETTER Upgrade font-awesome to version 5.

### CODE REVIEW

File: sonar/theme/assets/scss/common/_theme.scss [hunk 1]
```diff
@@ -1,10 +1,12 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap');
```
Comment (iGormilhit) [inline sonar/theme/assets/scss/common/_theme.scss]: I think we should host the font files too, for privacy concerns.

Comment (sebastiendeleze) [reply sonar/theme/assets/scss/common/_theme.scss]: you're right, thanks

**Comment by sebastiendeleze** [comment]
@jma You're right, invenio-theme uses Font Awesome 4 in several templates. I've overriden any of them, but not all.

I'll downgrade the library.

File: sonar/theme/templates/sonar/footer.html [hunk 2]
```diff
@@ -24,8 +25,22 @@
...
+      <div class="col-12 col-sm-6 py-2 py-sm-5 text-center text-sm-left">
+        <div class="d-flex flex-row">
+          <div>
+              <a href="#">{{_('Help')}}</a>
+          </div>
+          <div class="ml-3">
+              <a href="#">{{_('About')}}</a>
+          </div>
+          <div class="ml-3">
+              <a href="#">{{_('Contact')}}</a>
```
Comment (jma) [inline sonar/theme/templates/sonar/footer.html]: link to contact view, what about "About", "Help"?

Comment (sebastiendeleze) [reply sonar/theme/templates/sonar/footer.html]: It's just samples to see how links are displayed in footer. 
I can create views for each links, but there's no content to show on this pages at this time.
Is it better to have links without targets or links pointing to an empty page?

File: .gitignore [hunk 3]
```diff
@@ -25,6 +25,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.env
```
[the remark this answers is not in the document]
Comment (sebastiendeleze) [reply .gitignore]: Absolutely correct, this modification was present in PR #8. I'll remove it

File: sonar/theme/templates/sonar/footer.html (hunk 2 above)
Comment (jma) [reply sonar/theme/templates/sonar/footer.html]: Ok but it seems you have a view for contact:
https://github.com/rero/sonar/pull/9/files#diff-71846f390683019c64177dcb3c8b53caR29
no?

Comment (sebastiendeleze) [reply sonar/theme/templates/sonar/footer.html]: I removed contact view, but I have to keep views.py otherwise sonar templates are not found anymore

**Pull request merged**